projectroles/tests/test_views.py

```diff
@@ -5012,12 +5016,21 @@ def test_create_target(self):
         }
         model_dict = model_to_dict(site)
         self.assertEqual(model_dict, expected)
+
+        tl_event = ProjectEvent.objects.filter(
+            event_name='target_site_create'
+        ).first()
+        self.assertEqual(tl_event.event_name, 'target_site_create')
         with self.login(self.user):
             self.assertRedirects(response, reverse('projectroles:remote_sites'))
 
     @override_settings(PROJECTROLES_SITE_MODE=SITE_MODE_TARGET)
     def test_create_source(self):
         """Test creating a source site as target"""
+        tl_event = ProjectEvent.objects.filter(
+            event_name='source_site_set'
+        ).first()
+        self.assertIsNone(tl_event)
```

Comment:
You can simplify this further by using `count()` or `exists()` and assert true/false.